src/main/java/com/vinylteam/vinyl/service/impl/DefaultDiscogsService.java

```diff
@@ -0,0 +1,74 @@
+package com.vinylteam.vinyl.service.impl;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.vinylteam.vinyl.service.DiscogsService;
+import com.vinylteam.vinyl.discogs4j.client.DiscogsClient;
+import com.vinylteam.vinyl.discogs4j.entity.DiscogsVinylInfo;
+import com.vinylteam.vinyl.discogs4j.entity.RawResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class DefaultDiscogsService implements DiscogsService {
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    private final ObjectMapper objectMapper;
```

Comment:
Reminder that we agreed on an empty row after classes' '{' and before closing '}'.